tests/test_mega_hardware_serial/test_mega_hardware_serial.ino

```diff
@@ -127,6 +127,36 @@ test(2_junk_in_buffer) {
   
 }
 
+test(3_compliance_margins) {
+  test_setup();
+  motor.init();
+
+  // Intentional minor deviations from factory default for AX-12
+  byte cw_margin = 0x05;
+  byte ccw_margin = 0x03;
+  byte cw_slope = 0x44;
+  byte ccw_slope = 0x3A;
+
+  byte orig[4] = {0x0};
+  byte read[4] = {0x0};
+
+  assertTrue(motor.getComplianceMargins(orig[0], orig[1], orig[2], orig[3]));
```

Comment:
You caught me not following my own advice: I should have edited the test first, then edited the code so it would pass. Will fix.